.env

```
@@ -0,0 +1,6 @@
+HOST=localhost
+PORT=5432
+DATABASE=postgres
+USER=admin
+PASSWORD=admin
```

these should probably be prefixed with `DB`

```suggestion
DB_HOST=localhost
DB_PORT=5432
DATABASE=postgres
DB_USER=admin
DB_PASSWORD=admin
```